app/views/content/what-is-a-pgce.md

```diff
@@ -51,6 +51,8 @@ Getting QTS only means that you’re qualified to teach in most schools in Engla
 
 Getting PGCE only means you will not be qualified to teach in most primary, secondary and special schools in England, although some schools will employ teachers without QTS.
 
+You will also not be entitled to the same benefits during your training and career as teachers that have QTS. For example, better pay and support when you start teaching.
```

Comment:
Is it clear that this sentence refers to PGCE only @EllieNodder ? Appreciate that one big para on PGCE only wouldn't be good but if I was skimming this, I'm not sure whether I'd immediately pick up that this was about PGCE only?